content/2015.12.hacker-news.rst

```diff
@@ -0,0 +1,145 @@
+Hacker News answers
+###################
+
+:date: 2015-12-15
+:slug: hackernews-feedback
+:lang: en
+:url: hackernews-feedback
+:summary: Where we answer a set of questions that arised on a Hacker News
+          thread.
+
+Today, `Kinto was featured on Hackernews
+<https://news.ycombinator.com/item?id=10733164>`_. We take this as a great
+milestone for the project, even if it also comes with some sort of pressure and
+stress :)
+
+Since a few questions emerged from the thread there, we thought it'd
+appropriate to provide some answers here.
+
+What does this name mean?
+=========================
+
+Kinto-un is the name of the flying nimbus in Dragon ball. It's said that
+"the cloud can only be ridden by someone with a pure heart". Oh, well.
+    
+Maintaining a service is hard.
+==============================
+
+The main benefit of Firebase and Parse is that you don't have to run them.
+
+That is true. We want to make it very easy to deploy / use. We can even
+imagine to allow developers to run a Kinto instance in one-click on
+DigitalOcean, Heroku or Bitnami.
+
+Or if someone runs a public Kinto instance and wants to charge for its usage,
+that's completely fine!
+
+At Mozilla, we just can't rely on third parties like Firebase or Parse for
+critical data.
+
+If you read french, we've written `an article about our vision on self-hosting
+<http://www.servicedenuages.fr/difficile-promesse-autohebergement>`_
+(yes, we need to translate it!)
+
+How can you communicate with us?
+================================
+
+* Our IRC channel ``#kinto`` on ``irc.freenode.net``. You can `click here to start
+  a discussion
+  <https://www.irccloud.com/invite?channel=%23kinto&amp;hostname=irc.freenode.net&amp;port=6697&amp;ssl=1>`_
+  or join using your favorite client.
+* Our team blog http://www.servicedenuages.fr/
+* `The Kinto mailing list <https://mail.mozilla.org/listinfo/kinto>`_.
+
+Realtime updates, what about Kinto?
+===================================
+
+When a record is modified, Kinto sends a notification on a pubsub channel. We
+integrated Pusher.com and WebPush very recently.  Currently, the notification
+payload contains the old and new version of the record. We plan on making
+improvements, like sending diffs only. #babysteps #ftw! 
+      
+Also, the current mecanism does not allow to send write operations via
+websockets. Those are sent via the REST API.  
+
+You're missing an administration panel !
+========================================
+
+That is true, but `we're working on it pretty hard
+<https://github.com/Kinto/kinto-admin >`. If you want, you can have a look at
+it now, even though it's fairly new.
```

Comment:
Ready this make me feel kinto is not ready for me to use yet. Everything looks new and not ready.